atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -1083,46 +1078,23 @@ private static LockAndTimestampServices getLockAndTimestampServices(
         ServiceCreator creator = ServiceCreator.withPayloadLimiter(
```

Comment:
Is this thing used still here?

Reply:
Yeah, by `ConjureLockWatchingService`. I'll clear that out in a quick follow up to this one.